.travis.yml

```diff
@@ -32,9 +32,17 @@ matrix:
     - python: "3.6"
       env:
         - TOX_ENV=pythonbuild3.6
+    - python: 3.7
+      dist: xenial
+      sudo: true
```

Comment:
Yes, it's because it obtains Python 3.7 from some addition source and installs it. We cannot specify "Python 3.7" without these extra options. It's been the case for quite some time now, but Travis seems to need some time for releasing new images.